src/EnergyPlus/Coils/CoilCoolingDXCurveFitOperatingMode.cc

```diff
@@ -353,6 +353,13 @@ void CoilCoolingDXCurveFitOperatingMode::CalcOperatingMode(EnergyPlus::EnergyPlu
     }
     outletNode.Temp = Psychrometrics::PsyTdbFnHW(outletNode.Enthalpy, outletNode.HumRat);
```

Comment:
Why is line 354 hanging out here like this. It's already been calculated unless contfan is used. I am moving this line and the below code change into the contfan block just for my own santiy check.